index.js

```diff
@@ -388,6 +388,23 @@ instance.prototype.updateTransitionList = async function() {
 	self.init_feedbacks();
 }
 
+instance.prototype.updateFilterList = async function() {
+	var self = this;
+
+await self.obs.send('GetSourceFilters).then(data => {
```

Comment:
Needs a closing ' after Filters.

We also need to send it a source name as part of the request https://github.com/Palakis/obs-websocket/blob/4.9.0/docs/generated/protocol.md#getsourcefilters

Reply:
I've added it, not sure if it works that way though